README.md

```diff
@@ -43,7 +43,7 @@ Being against the so-called "Assistance and Access Bill" does not mean you are i
 
 We are against the bill because it is a destructive and shortsighted law. In addition, most of (if not all) the services that law enforcement want to target with this law are based outside of Australia. This law is effectively useless in forcing them to build backdoors or break encryption.
 
-In all good conscience, we can no longer support Labor. And we will be advocating for people to choose anyone but Labor.
+In all good conscience, we can no longer support Labor. And we will be advocating for people to choose anyone but Labor or the Coalition.
```

Comment:
Hey @tserong. Maybe let's broaden this to "advocating for people to choose anyone that protect digital rights."

https://theyvoteforyou.org.au/divisions/senate/2018-12-06/32